### ISSUE TITLE
add preferred fonts to Java/Flash translation utility

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/tasks#572]
Creating an issue here, since checking Unfuddle is no longer part of my workflow.

Requests for 'preferred font' support in Java/Flash translation utility:

Tigrinya: https://phet.unfuddle.com/a#/projects/9404/tickets/by_number/3704
Gujarati: https://phet.unfuddle.com/a#/projects/9404/tickets/by_number/3714
Tibetan: https://phet.unfuddle.com/a#/projects/9404/tickets/by_number/3719

@ariel-phet please let me know when you'd like these to be addressed.

**Comment by pixelzoom** [comment]
Mentioning @oliver-phet, since he created the Unfuddle tickets.

**Comment by phet-steele** [comment]
@pixelzoom @ariel-phet I do want to say that when you decide the priority of these, Tibetan is a recent request by a currently active translator who yearns to do Java/Flash :smile:

**Comment by ariel-phet** [comment]
@pixelzoom can you comment on a time estimate of what this would take?

**Comment by pixelzoom** [comment]
Hard to say how long it will take... The changes are relatively trivial, < 45 minutes.  What is almost always a problem is publishing a new version of Translation Utility, and a dev version of a sim that can be used (by the translator) to test it.  In the past, that has taken hours, depending on what state the Java/Flash build tools are in.  Then we need to get feedback from the translator - in the case of Tigrinya, that work was done 7 months ago, and we haven't received the feedback to proceed.

Also keep in mind that once the translator has verified that the new version of Translation Utility is published, that doesn't mean we're done.  The preferred fonts must be "poked" into every Java/Flash sim before any sims can actually be translated.

**Comment by ariel-phet** [comment]
@pixelzoom in that case, marking as low priority. I am guessing the "poking" gets done by @jonathanolson? If so, this is going to need to wait at minimum until the website server switch over is complete.

**Comment by ariel-phet** [comment]
@pixelzoom reassigning to you. In the next couple of months if a situation arises where you have a pause in other work and feel like taking a dive into this issue, feel free.

**Comment by pixelzoom** [comment]
@oliver-phet Could you please notify the translators that it will be possibly as long as "a couple of months" before we can fulfill their request?

**Comment by oliver-phet** [comment]
Translators have been notified.

**Comment by jonathanolson** [comment]
There's a part of the build process where this should be doable mostly automatically. If we've already told translators it will be a while, this would be great to use for testing to make sure this process will still work after we move to the new server.

**Comment by jonathanolson** [comment]
The build process resource deployment is tested and working (on our new server).

I'll be able to handle this after the move-over.

@pixelzoom, is the fonts file up-to-date in SVN?

**Comment by pixelzoom** [comment]
> @pixelzoom, is the fonts file up-to-date in SVN?

No - none of the work has been done for the above mentioned locales. Font names need to be verified, and dev versions need to be tested by the translators.

**Comment by ariel-phet** [comment]
@oliver-phet - I think we need to just drop support for anything new in Java-land. If you get further requests, just point people towards HTML5 and let them know that nothing new is being done with our legacy sims. Feel free to close out the related unfuddle issues.
